docs/source/ops/common-metrics.rst

```diff
@@ -207,3 +207,22 @@ daml_executor_tasks_queue_remaining
 - **Description**: Additional elements that this queue can ideally accept without blocking.
 - **Type**: Gauge
 - **Observation**: Only available for `type` = `thread_pool`
+
+Pruning metrics
```

Comment:
```suggestion
Pruning Metrics
```
(could be a capital, I'm never too sure .. better to check context from other docs )